src/QsCompiler/Transformations/SearchAndReplace.cs

```diff
@@ -230,15 +230,19 @@ public TypeTransformation(SyntaxTreeTransformation<TransformationState> parent)
             public override QsTypeKind OnUserDefinedType(UserDefinedType udt)
             {
                 var id = Identifier.NewGlobalCallable(new QsQualifiedName(udt.Namespace, udt.Name));
+#pragma warning disable 618 // UserDefinedType.Range is obsolete.
```

Comment:
After looking through all code, I realize that you maintain the full support for the old handles and they are still populated (that makes sense, thank you!). Regardless, we need to update everything within the compiler to work only with non-deprecated items, otherwise this will just haunt us once we remove the support for the deprecated handles.